specification/dartLangSpec.tex

```diff
@@ -11779,7 +11811,7 @@ \subsubsection{Statement Completion}
 then, unless otherwise stated, the execution of $s$ stops
 at that point and completes in the same way.
 \commentary{%
-For example, if execution of the body of a \DO{} loop returns a value,
+For example, if execution of the body of a \DO{} loop returns an object,
```

Comment:
Below there is an `\Q{... will be inserted here}`. If that is visible, make it a comment and a TODO. It's either there or not, it has no value to eventually be there.

Reply:
There is a TODO comment already, and the `\Q{...}` _is_ visible, so I just removed the `\Q{...}`.